webview/src/plots/components/ZoomedInPlot.tsx

```diff
@@ -60,13 +68,25 @@ export const ZoomedInPlot: React.FC<ZoomedInPlotProps> = ({
     }
   }, [])
 
-  const onNewView = () => {
+  const onNewView = (view: View) => {
     const actions: HTMLDivElement | null | undefined =
       zoomedInPlotRef.current?.querySelector('.vega-actions')
     if (!actions) {
       return
     }
 
+    appendActionToVega('SVG', actions, () => {
+      void view.toSVG().then(svg => {
+        const themedSvg = replaceThemeValuesForExport(svg, [
+          ThemeProperty.FOREGROUND_COLOR,
+          ThemeProperty.FONT
+        ])
+
+        const fullThemedSvg = preventSvgTruncation(themedSvg)
```

Comment:
[F] Without this we end up with SVGs that look like this: 

<img width="1153" alt="image" src="https://github.com/iterative/vscode-dvc/assets/37993418/54be3fb8-b444-4b38-a796-09842ee36c6a">

I have spent several hours looking for alternatives including things like https://github.com/tsayen/dom-to-image.

IMO this is good enough for now.